frontend/database/modules/datasets.js

```diff
@@ -553,6 +553,34 @@ const actions = {
       persistBackend: true,
     });
   },
+  async changeStatusToDefault(_, { dataset, records }) {
+    const newRecords = records.map((record) => ({
+      ...record,
+      selected: false,
+      status: "Default",
+    }));
+    const [currentStatus] = records.map((record) =>
+      record.status.toLowerCase()
+    );
+    const messageOk = `1 record is un${currentStatus} `;
+    const messageKo = `1 record could not be un${currentStatus} `;
+    try {
+      await _updateDatasetRecords({
+        dataset,
+        records: newRecords,
+        persistBackend: true,
+      });
+      Notification.dispatch("notify", {
+        message: messageOk,
+        type: "success",
+      });
+    } catch (err) {
+      Notification.dispatch("notify", {
+        message: messageKo,
+        type: "error",
+      });
```

Comment:
use a try/catch/finally instead of try catch => then in the try and catch update a message variable and a typeOfNotification variable => in the finally state call the notification component only once with the message and typeOfNotification variables